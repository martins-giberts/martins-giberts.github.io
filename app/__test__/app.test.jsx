/* globals describe, it, expect */

import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Component from '../App'

describe('App test', () => {
  it('should render without throwing an error', () => {
    const wrapper = shallow(<Component/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
