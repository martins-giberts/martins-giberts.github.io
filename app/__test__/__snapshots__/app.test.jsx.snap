// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test should render without throwing an error 1`] = `
<div
  className="app"
>
  <Header />
  <SocialLinks />
  <Footer />
</div>
`;
